app/javascript/components/community/video-grid/index.tsx

```diff
@@ -71,8 +72,8 @@ function VideoGridHeader({
         width={48}
         className="mr-24 self-start"
       />
-      <div className="grid gap-8 mr-auto">
-        <h2 className="text-h2">Learn from our community</h2>
+      <div className="mr-auto">
+        <h2 className="text-h2 mb-4">Learn from our community</h2>
```

Comment:
@dem4ron Grid seemed overkill to me so I simplified this to use a margin (and changed the margin as I think it's too big and I'll overriding Tai ;))